.github/CODEOWNERS

```diff
@@ -14,6 +14,7 @@ yarn.lock                                           @backstage/community-plugins
 /workspaces/analytics                               @backstage/community-plugins-maintainers  @jmezach
 /workspaces/announcements                           @backstage/community-plugins-maintainers  @kurtaking
 /workspaces/azure-devops                            @backstage/community-plugins-maintainers  @awanlin
+/workspaces/azure-sites                             @backstage/community-plugins-maintainers  @deepan10, @sarabadu
```

Comment:
@deepan10 @Sarabadu as people with Azure knowledge would you be open to sharing ownership of this plugin, if not that's totally fine 👍